src/App.js

```diff
@@ -8,19 +8,20 @@ function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <h1>
-          BOOKSTORE CMS
-        </h1>
-        <nav>
-          <ul>
-            <li>
-              <Link to="/categories">categories</Link>
-            </li>
-            <li>
-              <Link to="/">Books</Link>
-            </li>
-          </ul>
-        </nav>
+        <div>
+          <h1>BOOKSTORE CMS</h1>
+          <nav>
+            <ul>
+              <li>
+                <Link to="/categories">categories</Link>
```

Comment:
- [ ] The color of the Current Chart is different on design, and the name should be Current Chapter

Yours:
![image](https://user-images.githubusercontent.com/62705947/222691479-39b893c8-64a8-48f9-af01-189d273e29c4.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/222691436-5f575f00-0a9b-4413-8c8e-d4b043c5cbde.png)

Reply:
- [ ] The comments, remove and edit should have a | separating them

Yours:
![image](https://user-images.githubusercontent.com/62705947/222691722-60ef8562-8da0-45cb-aa66-ce4adc275a9f.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/222691768-70f41f49-ca93-4d21-892d-48e2b0488a6b.png)